pkg/table/tables/index.go

```diff
@@ -412,15 +412,12 @@ func (c *index) Delete(ctx table.MutateContext, txn kv.Transaction, indexedValue
 		// Only when pid in value equals to c.physicalID, the key can be deleted.
 		if c.idxInfo.Global {
```

Comment:
L413~L425 are useless now, which can be deleted. ref: https://github.com/pingcap/tidb/pull/52442?w=1

Reply:
Done, thx.